sycl/test-e2e/OptionalKernelFeatures/fp64-conv-emu-1.cpp

```diff
@@ -17,7 +17,13 @@
 // UNSUPPORTED: cuda, hip
 // UNSUPPORTED-INTENDED: FP64 emulation is an Intel specific feature.
 
-// RUN: %clangxx -fsycl -fsycl-targets=intel_gpu_dg2_g10,intel_gpu_dg2_g11,intel_gpu_dg2_g12,intel_gpu_pvc,intel_gpu_mtl_h,intel_gpu_mtl_u -fsycl-fp64-conv-emu %O0 %s -o %t.out
+// RUN: %clangxx -fsycl -fsycl-targets=intel_gpu_dg2_g10,intel_gpu_dg2_g11,intel_gpu_dg2_g12,intel_gpu_pvc,intel_gpu_mtl_h,intel_gpu_mtl_u -fsycl-fp64-conv-emu --no-offload-new-driver %O0 %s -o %t.out
+// RUN: %{run} %t.out
```

Comment:
Question: do we want to modify this test that way by adding 2 command lines with and without offload new driver? It looks like that way we would modify a lot of tests introducing command lines like that and then when we switch to new offload driver by default, we would need to clean up all that.
I'm thinking would not it be better to just run entire SYCL E2E with new offload driver enabled, and mark currently failing tests as XFAIL? and when we are ready, we would just flip the switch and stop new offload driver testing (since it would become a default)
That way we would not need to modify all tests like that...